CONTRIBUTING.md

```diff
@@ -316,3 +316,28 @@ PR paired with an e2e change (documented [here](https://github.com/redhat-appstu
 but the catalog PR pairing process will be complete.
 * Pairing is typically only required for breaking changes, so once the catalog PR is merged, things are now in a broken state. This should be resolved ASAP by
 marking the release-service PR as ready, merging it, and promoting its results to the infra-deployments overlays.
+
+### Data Keys Schema
+
+This repository maintains a json schema for the data key used in various tasks in this repo. It is stored [here](schema/dataKeys.json).
+
+If your change adds or removes a key to the data file, the schema must be updated accordingly as part of your pull request.
+
+### Checkton check
+
+This repository uses [checkton](https://github.com/chmeliik/checkton) to run shellchecker on the embedded shell in the tekton resources.
```

Comment:
It's called `shellcheck` , maybe you even could link it: https://www.shellcheck.net

Also, probably better to say "on the embedded scripts", and Tekton with capital T.

Reply:
fixed and linked